src/content/docs/guides/node-setup/deploy-a-node/index.mdx

```diff
@@ -125,15 +125,27 @@ rpm -i tenzir-static-amd64-linux.rpm
 </TabItem>
 <TabItem value="nix" label="Nix" icon="nix">
 
-Use our `flake.nix`:
+Use our `flake.nix` to run an ad-hoc tenzir process:
 
 ```bash
-nix run github:tenzir/tenzir/stable
+nix run github:tenzir/tenzir/latest
 ```
 
-Install Tenzir by adding `github:tenzir/tenzir/stable` to your flake inputs, or
+Or run a tenzir node without installing:
```

Comment:
If you refer to the product, use Tenzir Node. Otherwise just "node" and lowercase.